### ISSUE TITLE
Change facility - Unnecessary links after merging or migrating a super admin user 

### ISSUE THREAD

**Issue by pcenov** [learningequality/kolibri#10681]
## Observed behavior
After merging or migrating a super admin user that user is able to see and access the Permissions, Facilitates, Info and Settings pages in the Device plugin.

## Expected behavior
The Device plugin should be displayed with only a link to the Channels page.

## Steps to reproduce the issue
1. Install the latest develop build
2. Setup a device through the 'On my own' flow and migrate to a different device in the network

## Video

https://github.com/learningequality/kolibri/assets/79847249/149a0627-ebcd-43ea-8bec-0fa1016823b0

## Logs
[Logs.zip](https://github.com/learningequality/kolibri/files/11451971/Logs.zip)

## Usage Details
Ubuntu 22  - Chrome, Firefox

**Comment by pcenov** [comment]
@radinamatic

**Comment by rtibbles** [comment]
This is intended behaviour. A super user is still a super user after migrating. If our gherkins say otherwise, we should update them.

**Comment by pcenov** [comment]
Hi @rtibbles, just to make it crystal clear, my confusion here stems from the fact that after migrating the super user that super user has now actually lost access to the Coach and Facility plugins, but at the same time has full access to the Device plugin + there is the text 'Learn-only device Coach and admin features not available'. So could you double check that this is the correct behaviour and confirm it here?

**Comment by rtibbles** [comment]
Yes, this is intended. They have full access to the device plugin because they are still a super user, but we hide the coach and facility plugins because they do not function properly in this scenario.

**Comment by radinamatic** [comment]
And we also decided to live with the contradiction in the 'Learn-only device Coach and admin features not available' wording (since admin features for Device are still available), because we cannot have new or changed strings anymore?

**Comment by rtibbles** [comment]
coach and admin features are not available - superadmin features on device are still available.

**Comment by pcenov** [comment]
Closing as this is the intended behavior.
